e2e/qr-code-reader.e2e.ts

```diff
@@ -25,7 +27,9 @@ const testQRCode = async ({
   // Wait for video
   await page.waitForTimeout(2000);
 
-  await expect(page).toHaveScreenshot();
+  screenshotName !== undefined && screenshotName !== ""
+    ? await expect(page).toHaveScreenshot(screenshotName)
+    : await expect(page).toHaveScreenshot();
```

Comment:
Isn't passing undefined the same as passing nothing?

Reply:
I wish too but unfortunately no, that's why I add to add this